pom.xml

```diff
@@ -1445,7 +1445,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-pool2</artifactId>
-                <version>2.2</version>
+                <version>2.4.2</version>
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14560 (2.4.2)

Reply:
GeoServer 2.12.0 has version commons-pool-1.5.4.  (I.e., not comons pool2).

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16002